jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/tasklogs/TaskLogsViewImpl.java

```diff
@@ -18,43 +18,85 @@
 
 import java.util.List;
 import javax.enterprise.context.Dependent;
-import javax.enterprise.event.Event;
 import javax.inject.Inject;
 
-import com.google.gwt.safehtml.shared.SafeHtmlUtils;
-import com.google.gwt.user.client.ui.Composite;
+import org.gwtbootstrap3.client.ui.Button;
+import org.jboss.errai.common.client.dom.Div;
 import org.jboss.errai.common.client.dom.HTMLElement;
-import org.jboss.errai.common.client.dom.UnorderedList;
-import org.jboss.errai.ui.shared.api.annotations.DataField;
-import org.jboss.errai.ui.shared.api.annotations.Templated;
-import org.uberfire.workbench.events.NotificationEvent;
+import org.jboss.errai.common.client.dom.MouseEvent;
+import org.jboss.errai.databinding.client.api.DataBinder;
+import org.jboss.errai.databinding.client.components.ListComponent;
+import org.jboss.errai.ui.shared.api.annotations.*;
+import org.jbpm.workbench.common.client.util.AbstractView;
+import org.jbpm.workbench.ht.model.TaskEventSummary;
 
-import static org.jboss.errai.common.client.dom.DOMUtil.removeAllChildren;
-import static org.jboss.errai.common.client.dom.Window.getDocument;
+import static org.jboss.errai.common.client.dom.DOMUtil.addCSSClass;
+import static org.jboss.errai.common.client.dom.DOMUtil.removeCSSClass;
 
 @Dependent
 @Templated(value = "TaskLogsViewImpl.html")
-public class TaskLogsViewImpl extends Composite implements TaskLogsPresenter.TaskLogsView {
+public class TaskLogsViewImpl extends AbstractView<TaskLogsPresenter>
```

Comment:
@nmirasch I just noticed the components you're using here, they should be from the elmental2 package.

Reply:
@cristianonicolai sorry I didn't know that, I changed the TaskLogstViewImpl  components to the ones from elemental2 but for the ListComponent that we are currently using, only manages items that implements org.jboss.errai.common.client.api.IsElement, Do you think makes sense to migrate the item view to elemental2 too? Something similar to ListComponent that could work with that components?

Reply:
@nmirasch yep that's ok, ideally using elemental as much as possible but in this case there isn't an alternative.